core/src/nucleus/ribosome/api/mod.rs

```diff
@@ -489,6 +497,7 @@ pub mod tests {
             ("hc_entry_address", 12),
             ("hc_send", 13),
             ("hc_sleep", 14),
+            ("hc_remove_link", 15),
```

Comment:
Not suggesting a change rn, but I wonder if `hdk::remove_link` shouldn't be called `hdk::unlink_entries` to match better with `hdk::link_entries`?

Reply:
I think the "hc_" prefix comes from the hdk names rather than the functions that are being used internally? I could be wrong tho

Reply:
ooops didn't read that fully. I think it is definitely something to look into!

Reply:
Department of Names Department List of Names That Need To Be Considered:  DNDLNTNTBC!